Rakefile

```diff
@@ -1,5 +1,8 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
+require 'coveralls/rake/task'
```

Comment:
This content should be in a separate file `lib/tasks/ci.rake` and add the following content

```ruby
unless Rails.env.production?
  require 'rspec/core/rake_task'
  require 'cucumber/rake/task'
  require 'coveralls/rake/task'

  Coveralls::RakeTask.new

  namespace :ci do
    desc 'Run all tests and generate a merged coverage report'
    task tests: [:spec, :cucumber, 'coveralls:push']
  end
end
```

Once you do, make sure you can run the command `bundle exec rails ci:tests` and have both `RSpec` and `cucumber` running